### PR TITLE
[Samples] Consistent package names and directory structure

### PR DESCRIPTION
**Pull request by dturner**
The samples had a wild variety of package names. I've fixed that with the following steps: 

```
- Manually rename/move directories

Search and replace commands:

# Rename oboe.sample to oboe.samples
grep -rI "com\.google\.oboe\.sample\." -l . | xargs sed -i '' -e 's/com.google.oboe.sample./com.google.oboe.samples./g'

# Change JNI method names (mobileer)
grep -rI "com_mobileer_" -l . | xargs sed -i '' -e 's/com_mobileer_/com_google_oboe_samples_/g'

# Change com.google.sample to com.google.oboe.samples
grep -rI "com\.google\.sample\." -l . | xargs sed -i '' -e 's/com.google.sample./com.google.oboe.samples./g'

# Change samples.oboe to samples
grep -rI "samples\.oboe\." -l . | xargs sed -i '' -e 's/samples.oboe./samples./g'

# Change JNI method names (com_google_sample_oboe)
grep -rI "com_google_sample_oboe_" -l . | xargs sed -i '' -e 's/com_google_sample_oboe_/com_google_oboe_samples_/g'

# Change com.hyphonate.iolib 
grep -rI "com\.hyphonate\.iolib" -l . | xargs sed -i '' -e 's/com.hyphonate.iolib/com.google.oboe.samples.iolib/g'

# Change com.google.oboe.sample" to com.google.oboe.samples"
grep -rI "com\.google\.oboe\.sample\"" -l . | xargs sed -i '' -e 's/com.google.oboe.sample"/com.google.oboe.samples"/g'

# Change JNI method names (com_google_oboe_sample_)
grep -rI "com_google_oboe_sample_" -l . | xargs sed -i '' -e 's/com_google_oboe_sample_/com_google_oboe_samples_/g'

# Change com.mobileer to com.google.oboe.samples
grep -rI "com\.mobileer\." -l . | xargs sed -i '' -e 's/com.mobileer./com.google.oboe.samples./g'

# Change com.example.oboe. to com.google.oboe.samples
grep -rI "com\.example\.oboe\." -l . | xargs sed -i '' -e 's/com.example.oboe./com.google.oboe.samples./g'

# Change JNI method names (com_example_oboe_)
grep -rI "com_example_oboe_" -l . | xargs sed -i '' -e 's/com_example_oboe_/com_google_oboe_samples_/g'
```

### CODE REVIEW

File: samples/drumthumper/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ android {
     buildToolsVersion "29.0.1"
 
     defaultConfig {
-        applicationId "com.mobileer.drumthumper"
+        applicationId "com.google.oboe.samples.drumthumper"
```
Comment (philburk) [inline samples/drumthumper/build.gradle:9]: The use of com.mobileer in DrumThumper was deliberate because we plan to release it as an app on Play Store. We cannot use a package name of com.google in the Play Store for apps that are not first party.

File: samples/drumthumper/src/main/java/com/google/oboe/samples/drumthumper/DrumThumperActivity.kt [hunk 2]
```diff
@@ -26,18 +26,14 @@ import android.widget.Button
...
 import android.widget.Toast
-
 import androidx.appcompat.app.AppCompatActivity
-
-import java.util.*
-
-import java.time.LocalDateTime;
-
+import java.time.LocalDateTime
+import java.util.Timer
```
Comment (philburk) [inline samples/drumthumper/src/main/java/com/google/oboe/samples/drumthumper/DrumThumperActivity.kt:31]: Cleaning up imports is fine but should be a separate CL.

File: samples/drumthumper/build.gradle (hunk 1 above)
Comment (dturner) [reply samples/drumthumper/build.gradle:9]: Sure, but that doesn't mean we need to use `com.mobileer` as the default package name in the project. Doing so causes Android Studio to complain that all the classes have the wrong package name. What we should do is have a 'prod' config in gradle which changes the `applicationId` when we want to create an APK for Play upload. This should be done in a separate PR though.

Comment (philburk) [reply samples/drumthumper/build.gradle:9]: Don says the package names do not matter for Play Store. Lacking evidence to the contrary I will approve this.
@android-usb - What do you think?

Comment (android-usb) [reply samples/drumthumper/build.gradle:9]: I am inclined to trust Don on this.
